frontend/src/scenes/insights/InsightHistoryPanel/InsightHistoryPanel.tsx

```diff
@@ -0,0 +1,264 @@
+import React, { useState } from 'react'
+import { Tabs, Modal, Input, Button, List, Col, Spin } from 'antd'
+import { toParams, dateFilterToText } from 'lib/utils'
+import { Link } from 'lib/components/Link'
+import { PushpinOutlined, PushpinFilled, DeleteOutlined } from '@ant-design/icons'
+import { useValues, useActions } from 'kea'
+import { insightHistoryLogic } from './insightHistoryLogic'
+import { ViewType } from '../insightLogic'
+import { keyMapping } from 'lib/components/PropertyKeyInfo'
+import { formatPropertyLabel } from 'lib/utils'
+import { cohortsModel } from '~/models'
+import { PropertyFilter, Entity, CohortType } from '~/types'
+
+const InsightHistoryType = {
+    SAVED: 'SAVED',
+    RECENT: 'RECENT',
+}
+
+const { TabPane } = Tabs
+
+const determineFilters = (viewType: string, filters: Record<string, any>, cohorts: CohortType[]): JSX.Element => {
+    const result = []
+    if (viewType === ViewType.TRENDS) {
```

Comment:
Wonder if this entire section could look a touch prettier if it was in a table or some other format?

Reply:
done!

Reply:
Sorry this is nitpick-y but would look better without the `:`. It'd also be good to have the table be 100% width (or at least consistent) and have a tooltip over the "save" button.